browser_history/__init__.py

```diff
@@ -1,2 +1,31 @@
+import logging
+from . import generic
 from . import browsers
-from browser_history.utils import get_history
+
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(levelname)s: %(message)s')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
The namespace is getting polluted here.
This will make `browser_history.logger`, `browser_history.handler`, etc. visible which isn't desirable. The only relevant names needed under `browser_history` are `browsers` and `get_history`.

Reply:
Should I add it to a function/class then?